seed/utils/geocode.py

```diff
@@ -0,0 +1,118 @@
+# !/usr/bin/env python
+# encoding: utf-8
+
+import requests
+
+from django.conf import settings
+
+from django.contrib.gis.geos import GEOSGeometry
+
+
+def long_lat_wkt(state):
```

Comment:
This is a helper method I found myself using in two different spots, so I placed it here. Is this the appropriate place?

Reply:
After some thought, this is probably better off as a Serializer. Let me know if you agree or disagree. (I think `rest_framework` might have some features for GIS fields.)

Reply:
I'm not sure what this is doing, what does GEOSGeometry().wkt do?

Reply:
From my understanding, `GEOSGeometry()` initializes a geometry object with, in this case, a long_lat input held in the DB as a Well-Known Binary (WKB) that looks like this `0101000000000000000000F03F000000000000F03F`. The `.wkt` part returns the Well-Known Text (WKT) version that looks like 'Point(104.54 39.78)`.

All this info coming from here: https://docs.djangoproject.com/en/2.1/ref/contrib/gis/geos/#django.contrib.gis.geos.GEOSGeometry

Reply:
@danielbressan I left a comment in the code. Let me know if you think it should be rephrased or generally changed.